libraries/splunk_template.rb

```diff
@@ -60,6 +61,10 @@ def variables(args = nil)
         end
       end
 
+      def sensitive(args = nil)
```

Comment:
Removing as it wasn't needed. 
Also, adding the require as I know we have run into issues with that in the past.